packages/storage-plus/src/indexed_map.rs

```diff
@@ -129,17 +129,17 @@ where
     }
 
     // use prefix to scan -> range
-    pub fn prefix(&self, p: K::Prefix) -> Prefix<T> {
+    pub fn prefix(&self, p: K::Prefix) -> Prefix<Vec<u8>, T> {
```

Comment:
Okay, this Vec<u8> is the "legacy" approach?

Reply:
Yes. It's an internal thing, so totally warranted IMO.